swagger.yaml

```diff
@@ -721,13 +727,78 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
+  "/v1/users/{userGuid}/flextime": 
+    get:
+      tags:
+        - UsersRead
+      summary: 
+        Get the flextime balance for a user for a specified date. Total balance
+        is returned for the given date. Month balance is the balance for the
+        month of the given date. Values are returned only if the advanced time
+        tracking add-on is active.
+      operationId: Flextime_GetFlextime
+      parameters:
+        - name: userGuid
+          in: path
+          required: true
+          description: Id of the user.
+          schema:
+            type: string
+          x-position: 1
+          x-relation-guid: UserOutputModel
+        - name: eventDate
+          in: query
+          description:
+            Date for which to get the balance. Max 12 months into the future.
+          schema:
+            type: string
+            format: date-time
+          x-position: 2
+      responses:
+        "200":
+          description: FlextimeModel.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FlextimeModel"
+        "403":
+          description: Addon required
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "404":
+          description: Resource not found
+        default:
+          description: Default error response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
```

Comment:
It look like this is copied from the severa flexitime end point but this is not necessary. In our case we just want to provide the balance for the current day (and potentially month) in both cases this means no eventDate param is required,
Lets refer to userGuid as severaGuid as this is what is is called in out user schema.

Reply:
This is from Severa Docs: Endpoint get the flextime balance for a user for a specified date. Total balance is returned for the given date. Month balance is the balance for the month of the given date.
Im not sure if date is not given, will it use current date as default.